docs/concepts/crds/function/index.md

```diff
@@ -5,23 +5,23 @@ menu:
     identifier: function-overview
     name: Function
     parent: crds
-    weight: 30
+    weight: 35
 product_name: kubestash
 menu_name: docs_{{ .version }}
 section_menu_id: concepts
 ---
 
-> New to Stash? Please start [here](/docs/concepts/README.md).
+> New to KubeStash? Please start [here](/docs/concepts/README.md).
 
 # Function
 
 ## What is Function
 
-A complete backup or restore process may consist of several steps. For example, in order to backup a PostgreSQL database we first need to dump the database and upload the dumped file to a backend. Then we need to update the respective`Repository` and `BackupSession` status and send Prometheus metrics. In Stash, we call such individual steps a `Function`.
+A complete process of a task such as, backup or restore, is called a `Function` in KubeStash.
```

Comment:
In KubeStash, a complete task like backup or restore is referred to as a `Function`.